packages/rpc/src/tests/endpoint.test.ts

```diff
@@ -116,5 +118,27 @@ describe('createEndpoint()', () => {
         message: expect.stringContaining('terminated'),
       });
     });
+
+    it('avoid sending post message on a terminated endpoint', () => {
+      let releaseRef: (args?: any) => void;
+
+      const createEncoder = (options: EncodingStrategyApi) => {
+        releaseRef = options.release;
+        return createBasicEncoder(options);
+      };
+
+      const {port1} = new MessageChannel();
+      port1.start();
+
+      const messenger = fromMessagePort(port1);
+      const endpoint = createEndpoint(messenger, {createEncoder});
+
+      const spy = jest.spyOn(messenger, 'postMessage');
+
+      endpoint.terminate();
+      releaseRef!();
```

Comment:
Hmm it's a bit confusing what this value actually refers to, I admit I am not even sure myself without digging into the `createEncoder` option a bit. It feels like it's testing an implementation detail, though. I think it would be nicer if we could have a "proper" example where a function is actually proxied across the bridge, and is then used to call the global `release()` function provided by the library